cmd/erasure-multipart.go

```diff
@@ -579,13 +599,13 @@ func (er erasureObjects) PutObjectPart(ctx context.Context, bucket, object, uplo
 	// Must be held throughout this call.
 	partIDLock := er.NewNSLock(bucket, pathJoin(object, uploadID, strconv.Itoa(partID)))
 	plkctx, err := partIDLock.GetLock(ctx, globalOperationTimeout)
-	if err != nil {
-		return PartInfo{}, err
-	}
 	pctx := plkctx.Context()
 	defer partIDLock.Unlock(plkctx)
 
-	onlineDisks := er.getDisks()
+	partSuffix := fmt.Sprintf("part.%d", partID)
+	erPL := er.setPlacement(partSuffix)
```

Comment:
Question: if the placement depends only on part number, doesn't this mean there is one erasure set that will receive all of the part.1 ? (also I saw many applications that use multipart mechanism to upload only one part)

Reply:
Yes that doesn't change like single PUT uploads we can't really do anything about that.

Reply:
Let me make that change

Reply:
This has been fixed @vadmeste